File: JS/UberX.js
import Car from './Car.js'

export default class UberX extends Car {
  constructor(license, driver, brand, model) {
    super(license, driver)
    this.brand = brand
    this.model = model
  }

  printDataCar() {
    super.printDataCar()
    console.group()
    console.group('Carro:')
    console.log('Marca: ' + this.brand)
    console.log('Modelo: ' + this.model)
    console.groupEnd()
    console.groupEnd()
  }
}

File: Python/uberX.py
from car import Car

class UberX(Car):
  brand = str
  model = str

  def __init__(self, license, driver, brand, model):
    super().__init__(license, driver)
    self.brand = brand
    self.model = model

  def printDataCar(self):
    super().printDataCar()
    print("Marca: " + self.brand)
    print("Modelo: " + self.model)
